src/components/Questionnaire/QuestionTypes/DiagnosisQuestion.tsx

```diff
@@ -591,9 +592,17 @@ export function DiagnosisQuestion({
                       <CareIcon icon="l-times" className="h-5 w-5" />
                     </Button>
                   </div>
-                  <CommandList className="max-h-[100vh] overflow-y-auto pb-1">
-                    {diagnosisDetailsContent}
-                  </CommandList>
+                  <SheetContent
+                    side="bottom"
+                    className="h-[50vh] px-0 pt-2 pb-0 rounded-t-[10px]"
```

Comment:
use standard available roundedness values

```suggestion
                    className="h-[50vh] px-0 pt-2 pb-0 rounded-t-md"
```